coc/ext/fullwarapi/__init__.py

```diff
@@ -0,0 +1,221 @@
+"""An extension that helps interact with the FullWar API."""
+
+import asyncio
+import base64
+import logging
+import json
+
+from collections import namedtuple
+from datetime import datetime
+from typing import List
+
+import aiohttp
+
+from ...http import json_or_text
+from ...utils import correct_tag
+from ...wars import ClanWar
```

Comment:
`__init__` files are traditionally used to mark a package and to set up pathing not hold the source code for the package. Maybe we should update this by moving the source to its own file in this directory. What do you all thing?

Reply:
I agree, but I don't think this is very urgent so I would like to postpone it to the next release.

Reply:
same issue with discord links api then. I just copied the style